segment_path.go

```diff
@@ -135,12 +135,17 @@ func (pt *path) getAgnosterShortPath() string {
 	root := pt.rootLocation()
 	base := base(pt.env.getcwd(), pt.env)
 	pathDepth := pt.pathDepth(pt.getShortPath())
-	if pathDepth == 1 {
+	if pathDepth <= 1 {
```

Comment:
IMO, this was a bug with `agnoster_short`. It didn't match the documentation, which reads "When *more than 1 level deep*, it renders one folder_icon followed by the name of the current folder separated by the folder_separator_icon.". 

The previous code was displaying the folder_icon when the depth was 0. On windows, if the cwd was `C:\`, the path segment would display `C: > .. > \` instead of the expected `C: > \`

Reply:
This is failing tests on ubuntu, because [filepath.VolumeName](https://golang.org/pkg/path/filepath/#VolumeName) behaves differently on windows vs ubuntu. I'm going to revert this part of the PR, and address `agnoster_short` in another PR. I also believe it's acting unexpectedly on linux when you're at `/`.